examples/tfengine/generated/org_foundation/cicd/configs/import.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+DIRS=()
+
+while getopts "d:" c
+do
+  case $c in
+    d) DIRS="${OPTARG}" ;;
+    *)
+      echo "Invalid flag ${OPTARG}"
+      exit 1
+      ;;
+  esac
+done
+
+ROOT=$(realpath .)
+# Read DIRS from a space-separated string to list
+IFS=' ' read -r -a DIRS <<< "${DIRS}"
+
+for mod in "${DIRS[@]}"
+do
+    cd "${ROOT}"/"${mod}"
+    terraform init
+    terraform plan -out=plan.tfplan
+    project_id=$(terraform show -json plan.tfplan | jq -rM '.resource_changes[]? | select(.change.actions | index("create")) | select(.address | index("module.project.module.project-factory.google_project.main")) | .change.after.project_id')
+    if ! [[ -z "${project_id}" ]]; then
+      terraform import module.project.module.project-factory.google_project.main ${project_id}
```

Comment:
Can we use tfimport? Seems that would scale better.

Reply:
I was initially trying that. But it seems the --interactive=false mode does not working fully as expected (was able to skip something but not all), so turned back to a single import command for now. Also since it's going to be run as part of the automation, I would like to be able to control what type of resources I would like to import instead of going through the whole thing which could increase the chance of errors. I can open some issues for tfimport to fix the interactive issue as well support a type list argument and then use it here.

Reply:
Would it be useful to add a --resources flag that would allow a user to specify a list of resources to look at?

So then you could do tfimport --resources=google_project --nointeractive

Reply:
@MartinPetkov another data point for you. I tried both --nointeractive and --interactive=false, but it seems to stop at the lien resource

```
$ tfimport . --nointeractive
2021/02/04 18:01:47 Running: [terraform init]
2021/02/04 18:01:52 Running: [terraform plan -out plan-for-import-795646437.tfplan]
2021/02/04 18:01:56 Running: [terraform show -json plan-for-import-795646437.tfplan]
2021/02/04 18:01:58 Found importable resource: "module.project.module.project-factory.google_resource_manager_lien.lien[0]"
2021/02/04 18:01:59 Could not determine "name" automatically
2021/02/04 18:01:59 No existing liens found in 488909503971
Enter a value manually (blank to skip):

```

But if we can support a `--resources` array then, we can work around this problem as well.

Reply:
If --interactive=false doesn't work, that's definitely a bug. I'll fix that first.

Having a --resources flag should be possible. Let me look into it after.

Reply:
Done:
* https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/778
* https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/779

Reply:
Done